dataset1.py

```
@@ -0,0 +1,111 @@
+    # 주의 : 검증 및 테스트 과정에 반드시 최근 10일 데이터를 사용해야 하므로 수정하지 말 것
+    training_x, test_x = x[:-10], x[-10:]  # !!! 수정 금지 !!!
+    training_y, test_y = y[:-10], y[-10:]  # !!! 수정 금지 !!!
+
+    
+    return (training_x, training_y) if is_training else (test_x, test_y)
+
+
+
+def merge_data(symbols, data_dir='data'):
+
+    dates = pd.date_range(start_date, end_date, freq='D')
```

Ruff (0.12.2)

Undefined name `start_date`

(F821)